src/Public/js/myradio.webcam.tracker.js

```diff
@@ -8,6 +8,10 @@ function webcamTrackViewer() {
     type: "get",
     cache: false,
     url: myradio.makeURL("Webcam", "a-trackViewer"),
+    statusCode: {
+      // API returns 400 when timedelta is too big (minimised tab or whatever) clientside should ignore.
+      400: function () {}
```

Comment:
This is probably good for now, but maybe raise an issue to use the Page Visibility API to stop sending updates?